modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/CreationMenu.js

```diff
@@ -22,13 +22,70 @@ import React, {useContext, useState} from 'react';
 import FrontendDataSetContext from '../../FrontendDataSetContext';
 import {triggerAction} from '../../utils/actionItems/index';
 
-function CreationMenu({inEmptyState, primaryItems}) {
+const EMPTY_STATE_BUTTON_PROPS = {
+	'aria-label': null,
+	'className': null,
+	'displayType': 'secondary',
+	'title': null,
```

Comment:
Is `null` right for jsx props? I would think an explicit `undefined` might be better. I primarily wasn't sure if the html would render like `aria-label="null"`

Reply:
Yeah, you're right. Technically it works with `null`, but there is likely some hacky fix for that somewhere downstream. TS would not like this, I tried:
```
Type '{ children: Element; className: string; "aria-label": null; }' is not assignable to type 'DetailedHTMLProps<ButtonHTMLAttributes<HTMLButtonElement>, HTMLButtonElement>'.
  Type '{ children: Element; className: string; "aria-label": null; }' is not assignable to type 'ButtonHTMLAttributes<HTMLButtonElement>'.
    Types of property ''aria-label'' are incompatible.
      Type 'null' is not assignable to type 'string | undefined'
```
I'll change it!